yarn-project/sequencer-client/src/sequencer/public_processor.test.ts

```diff
@@ -708,13 +653,13 @@ describe('public_processor', () => {
       const txEffect = toTxEffect(processed[0]);
       expect(arrayNonEmptyLength(txEffect.publicDataWrites, PublicDataWrite.isEmpty)).toEqual(3);
       expect(txEffect.publicDataWrites[0]).toEqual(
-        new PublicDataWrite(computePublicDataTreeLeafSlot(baseContractAddress, contractSlotA), fr(0x102)),
+        new PublicDataWrite(computePublicDataTreeLeafSlot(baseContractAddress, contractSlotC), fr(0x201)),
```

Comment:
The order has changed because we now concat data properly in removeRedundantPublicDataWrites in abstract_phase_manager.ts.